client/src/app/site/pages/meetings/pages/motions/modules/change-recommendations/services/motion-diff.service/motion-diff.service.ts

```diff
@@ -248,15 +248,25 @@ export class MotionDiffService {
             .replace(/\s+<\/DIV>/gi, `</DIV>`)
             .replace(/\s+<\/LI>/gi, `</LI>`);
         html = html.replace(/\s+<LI>/gi, `<LI>`).replace(/<\/LI>\s+/gi, `</LI>`);
+        html = html.replace(/<LI>&nbsp;/gi, `<LI>`);
+
+        // Remove closing tag of span for diff algorithm
+        html = html.replace(
+            /(<SPAN[^>]+os-line-number[^>]+?>)((\s*|&nbsp;)?<\/SPAN>)/gi,
+            (found: string, _span: string, _whitespace: string, _spanClose: string): string => _span
+        );
+        html = html.replace(/<P>&nbsp;/gi, `<P>`);
+        html = html.replace(/<BR>&nbsp;/gi, `<BR CLASS="os-line-break">`);
+        html = html.replace(/ CLASS=" "/gi, ``).replace(/ CLASS=""/gi, ``);
 
         html = html.replace(/\u00A0/g, ` `); // replace no break space
         html = html.replace(/\u2013/g, `-`);
         html = DomHelpers.replaceHtmlEntities(html);
 
         // Newline characters: after closing block-level-elements, but not after BR (which is inline)
         html = html.replace(/(<br *\/?>)\n/gi, `$1`);
-        html = html.replace(/[ \n\t]+/gi, ` `);
-        html = html.replace(/(<\/(div|p|ul|li|blockquote>)>) /gi, `$1\n`);
+        html = html.replace(/(\n\t)+/gi, ``);
+        html = html.replace(/(<\/(div|p|ul|li|blockquote>)>) /gi, `$1`);
```

Comment:
This might cause problems as tabstops and newlines are parsed as spaces in browsers and do not always need to occur together. 
Also with you change for example `\n\n\t\n\n\t\n` would now stay as it is instead of getting replaced by one space.